### PR TITLE
Install Storybook & Tailwind

### PR DESCRIPTION
**Pull request by CRBroughton**
## Description

Will require the previous PR merged in as it's based from that branch.

Installs Storybook, TailwindCSS and CVA to manage classes. I've also included a demo button component to show Tailwind properly working with Storybook.

### CODE REVIEW

**Comment by vallezw** [comment]
There are some merge conflicts @CRBroughton

**Comment by CRBroughton** [comment]
Should all be fixed and ready to merge :partying_face:

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (vallezw) [inline .vscode/settings.json:1]: Id vote for putting this file in the gitignore or is there any specific reason for including it?

**Review by vallezw — CHANGES_REQUESTED** [review]
Looks good to me I removed some of the boilerplate stuff, and left a nit comment. Look it over if you have the time

File: .vscode/settings.json (hunk 1 above)
Comment (CRBroughton) [reply .vscode/settings.json:1]: Is required for some of the tailwind support in VSCode

Comment (vallezw) [reply .vscode/settings.json:1]: Alright fair point we ll leave it in

**Pull request merged**